packages/block-library/src/navigation/edit/unsaved-inner-blocks.js

```diff
@@ -120,17 +165,17 @@ export default function UnsavedInnerBlocks( {
 
 	return (
 		<div className="wp-block-navigation__unsaved-changes">
-			<Disabled
+			<div
```

Comment:
Do we still want this to be disabled in some case?

Reply:
Updated so it renders a div or `Disabled` component if saving.